gin/sentrygin.go

```diff
@@ -52,9 +126,21 @@ func (h *handler) handle(ctx *gin.Context) {
 		hub = sentry.CurrentHub().Clone()
 	}
 	hub.Scope().SetRequest(ctx.Request)
+
+	transaction := sentry.StartTransaction(
+		ctx, fmt.Sprintf("%v %v", ctx.Request.Method, ctx.FullPath()),
+		sentry.ContinueFromHeaders(h.getTraceIDFromRequest(ctx), h.getBaggageFromRequest(ctx)),
+	)
```

Comment:
We'll also need to set a source, `sentry.TransctionSource(sentry.SourceURL),`.
Please construct the options above and pass them in.